### DISCUSSION TITLE
Firmware Selector website is down.

### DISCUSSION THREAD

**Discussion by LightMoon** [immortalwrt/immortalwrt#1261] [General]
### Describe the bug

The Firmware Selector website is down.

### ImmortalWrt version

n/a

### ImmortalWrt release

n/a

### ImmortalWrt target/subtarget

n/a

### Device

n/a

### Image kind

Official downloaded image

### Steps to reproduce

Browsing [ImmortalWrt Firmware Selector](https://firmware-selector.immortalwrt.org/) will result in getting 500 Internal Server Error.

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
fixed

**Reply by jinkazph** [reply]
down again.

**Reply by luisFPCosta** [reply]
> fixed

Down again.

**Reply by emadalsbri** [reply]
still down

**Reply by matt2k7** [reply]
the following is a copy paste from Immortalwrt's telegram page: 

"Bad News 1: The immortalwrt.kyarucloud.moe mirror site has suspended due to regulatory reasons. The asu and rsync servers are also affected.
Bad News 2: The hard drive of the ImmortalWrt main site has crashed.

The main site will be rebuilt within the next 7 days, and the default rsync server will be pointed to it. asu will be back online after the main site is rebuilt.

Please use mirrors from https://help.mirrorz.org/immortalwrt/ instead while we're working on it."
